plugin_test.go

```diff
@@ -74,15 +75,71 @@ func TestGetHelmCommand(t *testing.T) {
 }
 
 func TestResolveSecrets(t *testing.T) {
-	tag := "v0.1.1"
-	api := "http://apiserver"
-	token := "12345"
-	account := "helm"
-	os.Setenv("MY_TAG", tag)
-	os.Setenv("MY_API_SERVER", api)
-	os.Setenv("MY_KUBERNETES_TOKEN", token)
-	os.Setenv("MY_SERVICE_ACCOUNT", "helm")
 
+	testEnvs := []struct {
+		prefix  string
+		tag     string
+		api     string
+		token   string
+		account string
+	}{
+		{prefix: "PROD", tag: "v0.1.1", api: "http://apiserver", token: "12345", account: "helm"},
+		{prefix: "STAGING", tag: "12345678", api: "http://apiserver", token: "12345", account: "helm"},
```

Comment:
We had errors when tag only consisted of numbers, added this test case... but couldn't reproduce the error